src/main/webapp/app/entities/event-attendee/event-attendee-sort-modal.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Modal, ModalBody, ModalHeader } from 'reactstrap';
+import { Translate } from 'react-jhipster';
+import { eventAttendeeSortButtonList } from 'app/shared/util/event-attendee-sort.constants';
+import { EventAttendeeSortButton } from 'app/entities/event-attendee/event-attendee-sort-button';
+
+export interface IEventAttendeeSortButtonInfo {
+  buttonName: string;
+  buttonTranslateKey: string;
+  buttonSortProp: string;
+  buttonOrderProp: string;
+}
+
+export interface IEventAttendeeSortModalProps {
+  isOpen: Boolean;
```

Comment:
the `Boolean` type in typescript should be `boolean` (small capital b)

not sure this (capital letter version) will cause compilation error or not

Reply:
Resolve by https://github.com/Five-Fishes/Club-Management-React-Client/pull/57/commits/91ad3f60f40383ef447cd55a57334d19e731546d